app/assets/javascripts/discourse/app/components/invite-link-panel.js

```diff
@@ -0,0 +1,122 @@
+import I18n from "I18n";
+import Component from "@ember/component";
+import Group from "discourse/models/group";
+import { alias } from "@ember/object/computed";
+import discourseComputed from "discourse-common/utils/decorators";
+import Invite from "discourse/models/invite";
+
+export default Component.extend({
+  inviteModel: alias("panel.model.inviteModel"),
+  userInvitedShow: alias("panel.model.userInvitedShow"),
+  isAdmin: alias("currentUser.admin"),
+  maxRedemptionAllowed: 5,
+  inviteExpiresAt: moment()
+    .add(1, "month")
+    .format("YYYY-MM-DD"),
+
+  @discourseComputed
+  maxRedemptionOptions() {
+    return [
+      { id: 5, name: 5 },
+      { id: 10, name: 10 },
+      { id: 50, name: 50 },
+      { id: 100, name: 100 },
+      { id: 500, name: 500 },
+      { id: 1000, name: 1000 },
+      { id: 5000, name: 5000 },
+    ];
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+
+    this.reset();
+  },
+
+  @discourseComputed(
+    "isAdmin",
+    "inviteModel.saving",
+    "inviteModel.details.can_invite_to"
+  )
+  disabled(isAdmin, saving, can_invite_to) {
+    if (saving) return true;
+
+    if (!isAdmin) {
```

Comment:
everything else is on one line and this if is done on 3 ? :p

Reply:
copy./paste turned bad 😛